components/base/TabBarPanel.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import classnames from "../../modules/classnames";
+import { GREY_1, GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
+import { BOX_SHADOW_7 } from "../shadowDepths";
+import { ROBOTO } from "../typography";
+
+// Styles
+const Container = styled.div`
+  max-width: auto;
+  border-radius: 5px;
+  background-color: ${WHITE_PRIMARY};
+  :hover {
+    box-shadow: ${BOX_SHADOW_7};
+  }
```

Comment:
Having a hover state on this entire container looks off to me since it'll take up a pretty large portion of the page in practice

Reply:
I agree. I went off of the mocks, but I think we can skip the hover state for the container. 👍